packages/strapi-plugin-content-manager/admin/src/components/SelectMany/SortableList.js

```diff
@@ -0,0 +1,36 @@
+/**
+ *
+ * SortableList
+ *
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import cn from 'classnames';
+import { SortableContainer } from 'react-sortable-hoc';
+import SortableItem from './SortableItem';
+// CSS.
+import styles from './styles.scss';
+
+const SortableList = SortableContainer(({ items, onClick, onRemove }) => {
+  const shadowList = (items.length > 4 ? <div className={styles.sortableListLong}></div> : '');
+
+  return (
+    <div className={cn(styles.sortableList)}>
+      <ul>
+        {items.map((item, index) => (
+          <SortableItem key={`item-${index}`} index={index} sortIndex={index} item={item} onRemove={onRemove} onClick={onClick} />
```

Comment:
You should use something like an id as `key` for the `SortableItem` see this [article](https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318)